nix/route53-recordset.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, uuid, name, ... }:
+
+with lib;
+
+{
+
+  options = {
+
+    name = mkOption {
+      default = "nixops-${uuid}-${name}";
+      type = types.str;
+      description = "Name of the recordset.";
+    };
+
+    region = mkOption {
+      type = types.str;
+      description = "AWS region.";
+    };
```

Comment:
This option is currently not used as it is only required for latency-based resource record sets.

Reply:
@lo1tuma If the option is not used, should it be removed?

Reply:
We've not implemented all potential behaviours for this resource. Just the ones they need at HolidayCheck. AFAICR other features do need to have a region specified.

Reply:
As @moretea said, for the current feature set it is not needed. There are some route53 features that require a region. For now I think we can just remove it or at least make it optional.